qCC/ccGraphicalTransformationTool.cpp

```diff
@@ -604,6 +638,22 @@ void ccGraphicalTransformationTool::advRefAxisRadioToggled(bool state)
 	}
 }
 
+void ccGraphicalTransformationTool::incrementalTranslationToggle() {
```

Comment:
'{' on the next line

Reply:
It's been a while since my last commit, I forgot CC coding style :(.